SConstruct

```diff
@@ -7,6 +7,16 @@ import tarfile
 import zipfile
 from SCons.Script.SConscript import SConsEnvironment
 
+# hack to let this work with Python 2.6
```

Comment:
I know you copied this comment from elsewhere, could you change it to

    # Compatibility function for Python 2.6.